Readme.md

```diff
@@ -3,29 +3,27 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E2055)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
-
-* [Form1.cs](./CS/WindowsApplication1/Form1.cs) (VB: [Form1.vb](./VB/WindowsApplication1/Form1.vb))
-* [Program.cs](./CS/WindowsApplication1/Program.cs) (VB: [Program.vb](./VB/WindowsApplication1/Program.vb))
-<!-- default file list end -->
-# How to disable a particular row in the GridView
-
-## Starting with version 19.2 we supported this functionality out of the box via the  [Disabled Cell Behavior](https://docs.devexpress.com/WindowsForms/401146/common-features/behaviors/disabled-cell-behavior?v=19.2)
-<p>This example demonstrates how to enable/disable rows using the new behavior. A disabled row is a row that has specific appearance, and its in-place editors can't be activated.
-</p>
-
-
 
+# Data Grid for Windows Forms - How to disable a particular row in a Grid Control
 
+This example uses the [Disabled Cell Behavior](https://docs.devexpress.com/WindowsForms/401146/common-features/behaviors/disabled-cell-behavior) (part of the [BehaviorManager](https://docs.devexpress.com/WindowsForms/DevExpress.Utils.Behaviors.BehaviorManager) component) to disable individual rows in a [Grid Control](https://docs.devexpress.com/WindowsForms/DevExpress.XtraGrid.GridControl). 
 
+Disabled rows are painted with grayed out appearance settings. In-place editors cannot be activated for these rows.
```

Comment:
`Disabled rows are painted with grayed out appearance settings.`

Is it necessary to say "appearance settings"? For example, can we say the following?

`Disabled rows are grayed out.`

Reply:
```suggestion
Disabled rows are grayed out. In-place editors cannot be activated for these rows.
```